third_party/caja/html-sanitizer.js

```diff
@@ -1608,7 +1608,7 @@ var html = (function(html4) {
 
   var ATTR_RE = new RegExp(
     '^\\s*' +
-    '([-.:\\w]+)' +             // 1 = Attribute name
+    '(\\[?[-.:\\w]+\\]?)'  +    // 1 = Attribute name
```

Comment:
Small nit: This would allow names like `[attr` and `attr]`. We should instead use a pipe: `(\\[[-.:\\w]+\\]|[-.:\\w]+)`